rewriter/Minitest.cc

```diff
@@ -145,7 +178,42 @@ string to_s(core::Context ctx, unique_ptr<ast::Expression> &arg) {
     return arg->toString(ctx);
 }
 
-unique_ptr<ast::Expression> runSingle(core::MutableContext ctx, ast::Send *send) {
+// The 'context' here is any expressions which are put at the top of the 'it' blocks. If we've looked at an instance of
+// `test_each`, then we need to somehow pull the variable being introduced by the `each` into the body of the method. We
+// do that by taking
+//
+//   test_each(expr) do |var|
+//
+// and turning it into an assignment like
+//
+//   var = T.must(expr.collect.first)
+//
+// which will have both the correct type and the correct name. (The reason for the `collect` is that if `expr` is an
```

Comment:
When the constant is un-annotated, it will be `T.untyped`. To me this seems no different than generating

```ruby
var = T.unsafe(nil)
```